block/block.go

```diff
@@ -20,7 +22,6 @@ func (m *Manager) applyBlock(ctx context.Context, block *types.Block, commit *ty
 	// TODO: add switch case to have defined behavior for each case.
 	// validate block height
 	if block.Header.Height != m.store.NextHeight() {
-		m.logger.Error("Block not applied. wrong height", "block height", block.Header.Height, "expected height", m.store.NextHeight())
 		return types.ErrInvalidBlockHeight
 	}
```

Comment:
@mtsitrin I'm curious about how to handle this error because I think it changed in your recent refactor

>I was just wondering, if there are any legitimate cases where the caller of applyBlock might get this error back and discard it, or if it should always be treated as a problem?

Reply:
resolved on huddle 👍